docs/source/main_classes/callback.mdx

```diff
@@ -66,8 +64,6 @@ Here is the list of the available [`TrainerCallback`] in the library:
 [[autodoc]] integrations.MLflowCallback
     - setup
 
-[[autodoc]] integrations.AzureMLCallback
```

Comment:
as a result of the below feedback, you probably can't get rid of this, but the messages around it might nudge a user to the mlflow callback